crates/cli/src/fs.rs

```diff
@@ -30,19 +30,13 @@ pub fn get_all_folders(root: &std::path::PathBuf) -> Vec<String> {
         if entry.file_type().is_dir() || entry.file_type().is_symlink() {
             // We need this mutation to ensure that both Unix and Windows paths resolves the same.
             // TODO: Find a better way to do this?
-            let simple_path = entry_path
-                .strip_prefix(root)
-                .unwrap()
-                .components()
-                .map(|c| c.as_os_str().to_string_lossy().to_string())
-                .collect::<Vec<_>>()
-                .join("/");
-
-            if !simple_path.is_empty() {
-                files.push(simple_path);
+            let simple_path = entry_path.strip_prefix(root).unwrap();
+            if !simple_path.as_os_str().is_empty() {
+                files.push(simple_path.to_string_lossy().to_string());
```

Comment:
Does OsString produce the same string for both unix and windows? If yea, simple_path.as_os_string() is needed here.

Reply:
yes, OsString produces the same string representation for both Unix and Windows systems. Thank you for pointing that out. I'll update the code to use simple_path.as_os_str() directly, which will ensure a more platform-independent approach. This change should contribute to better cross-platform compatibility